webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -192,6 +195,20 @@ const NavigationSideBar: React.FC = () => {
     updateSettings({ navbarCollapsed: !settings.navbarCollapsed });
   }, [settings.navbarCollapsed, updateSettings]);
 
+  useEffect(() => {
+    const keyDownListener = (e: KeyboardEvent) => {
+      if (e.code === 'KeyU' && (e.ctrlKey || e.metaKey)) {
```

Comment:
**q**: Why should this be Ctrl/Cmd + U instead of just picking one?
**fix**: On Windows Firefox ctrl+U opens up the page's source code, on Mac Firefox cmd+U does it. Whatever you decide for the shortcut, we should try to prevent that.
**sugg**: Since our goal is to eventually let these shortcuts be remapped, it might be nice to make this a little more generic to prepare for that.

Reply:
Per the ticket this should be configurable via user settings (UI is another ticket).